build/docker-compose.yml

```
@@ -43,7 +43,7 @@ services:
         release: "2022.2.1"
     depends_on:
       - base-notebook
-    image: ghcr.io/dask-notebook:latest
+    image: ghcr.io/dask/dask/dask-notebook:latest
```

One too much here :D

```suggestion
    image: ghcr.io/dask/dask-notebook:latest
```